ts/nni_manager/rest_server/restHandler.ts

```diff
@@ -98,8 +98,11 @@ class NNIRestHandler {
 
     private version(router: Router): void {
         router.get('/version', async (req: Request, res: Response) => {
-            const version = await getVersion();
-            res.send(version);
+            getVersion().then((version) => {
+                res.send(version)
+            }).catch((err: Error) => {
```

Comment:
This getVersion() function is used in trainingService, should fix package.json issue to get correct version number, not just catch error here.

Reply:
A note from the discussion:
Find a standard error handler as the fallback for all REST APIs.
Specific errors can still be handled at first place.